app/views/cdo/manage/tasks/record-microchip-deadline.njk

```diff
@@ -0,0 +1,74 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% extends '_layout.njk' %}
+
+{% block pageTitle %}When will the dog be fit to be microchipped? - {{ serviceName }}{% endblock %}
+
+{% block beforeContent %}
+  <div class="govuk-grid-row">
```

Comment:
Do you need a 'super()' in here to retain the Beta banner?

Reply:
good catch, did it with the other ones, but forgot for this